config/install/media_entity.bundle.twitter.yml

```diff
@@ -0,0 +1,21 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - media_entity_twitter
+id: twitter
+label: Twitter
```

Comment:
I have modified just the label. I think it should be better to keep the id as twitter as it should be more intuitive for anyone to search by "twitter". All the file names would be changed to tweet if we change the id. That's why I think we should keep the id as twitter. Let me know if you think otherwise.